test/test_utmpx.rs

```diff
@@ -0,0 +1,45 @@
+use nix::utmpx::*;
+
+/// Test opening and rewinding the DB.
+#[test]
+fn test_utmpx_open_rewind() {
+    let _m = ::UTMPX_MTX
+        .lock()
+        .expect("Mutex got poisoned by another test");
+    let mut db = Utmp::open().unwrap();
+    db.rewind();
+}
+
+/// Test iterating through default DB.
+#[test]
+fn test_iter() {
+    use std::io::Write;
+    let _m = ::UTMPX_MTX
+        .lock()
+        .expect("Mutex got poisoned by another test");
+    let mut db = Utmp::open().unwrap();
+
+    let mut entries = 0u64;
+    let mut found_booted = false;
+    for line in db.entries() {
+        entries += 1;
+        if let Ok(entry) = line {
+            if entry.entry_type() == EntryType::BootTime {
+                found_booted |= true;
+            }
+        }
+    }
+
+    // Cross-jobs in Travis seems to have an empty DB.
```

Comment:
I've tried a few combinations of target-conditions (the one quoted here was actually failing in CI), this seems to affect all Travis test-runs (including x86_64), I suspect it may be related to a containerized/virtualized step somewhere.
In additional tests, the annotation suggested in https://github.com/nix-rust/nix/blob/master/CONTRIBUTING.md#disabling-a-test-in-the-ci-environment did not really work to ignore it here.

I've temporarily put an `ignore` annotation here with a FIXME note. The tests itself should be fine (it works locally), but the CI needs some tweaks on Travis side to cope with it.